app/views/users/show.html.haml

```diff
@@ -0,0 +1,40 @@
+%section.user.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %article
+        %header
+          %img(src="http://www.gravatar.com/avatar/#{Digest::MD5.hexdigest('#{@user.email}'.downcase)}?s=40")
+          %h2= @user.full_name + "'s video collections (" + @user.queue_items.count.to_s + ")"
+          -if @user.followers.include?(current_user)
+            = link_to "Unfollow", unfollow_path(user_id: @user.id), method: :delete, class: "btn btn-default pull-right"
+          -else
+            = link_to "Follow", follow_path(user_id: @user.id), method: :post, class: "btn btn-default pull-right"
+          .clearfix
+        %table.table
+          %thead
+            %tr
+              %th(width="30%") Video Title
+              %th(width="15%") Genre
+          %tbody
+            - @user.queue_items.each do |item|
+              %tr
+                %td
+                  = link_to item.video.title, video_path(item.video)
+                %td
+                  = link_to item.video.category.name, categories_path(item.video.category)
+%section.user_reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %header
+        %h3= @user.full_name + "'s Reviews (" + @user.reviews.count.to_s + ")"
```

Comment:
If you use interpolation instead you have the built-in bonus that `@user.full_name` being nil doesn't blow up your app. Currently you would have a `NoMethodError '+' for Nil`.